.ci/dynamic-backstop-json-urls.js

```diff
@@ -3,16 +3,44 @@ var fs = require('fs');
 // Stash the directory where the script was started from
 var rootPath = process.cwd();
 // Get the contents of the backstop.json template
-var fileContents = require(rootPath + '/.ci/backstop.json');
+var fileContents = require(rootPath + '/.ci/backstop.default.json');
+
+// Relative URLs don't start with http
+function isRelativeURL(url) {
+  return !url.startsWith('http');
+}
+
+// Stash live URL, removing any trailing slash
+var liveURL = process.env.LIVE_SITE_URL.replace(/\/$/, "");
+
+// Stash multidev URL, removing any trailing slash
+var multidevURL = process.env.MULTIDEV_SITE_URL.replace(/\/$/, "");
 
 // Loop through all scrnarios in the template
-var newScenarios = fileContents.scenarios.map(function(scenario) {
-    // Set url of the scenarios to the multidev URL
-    scenario.url = process.env.MULTIDEV_SITE_URL;
-    // Set reference url of the scenarios to the live URL
-    scenario.referenceUrl = process.env.LIVE_SITE_URL;
```

Comment:
Previously this set all scenario URLs to the homepage. Not good for checking other pages!